cupy/indexing/scatter.py

```diff
@@ -0,0 +1,28 @@
+def scatter_update(a, indices, v, axis=0):
+    """Replaces specified elements of an array with given values.
+
+    ``v`` needs to be broadcastable to shape
+        ``a.shape[:axis] + indices.shape + a.shape[axis+1:]``.
```

Comment:
1. I added your suggestion.
2. I added similarity with `numpy.ndarray.__setitem__` to the doc because `numpy.put` does not take `axis` as an argument. Is it OK?
3. I added another example.